api/serializers/assets.py

```diff
@@ -471,18 +471,7 @@ class AssetIncidentReportSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = models.AssetIncidentReport
-        fields = (
-            "id",
-            "asset",
-            "incident_type",
-            "incident_location",
-            "incident_description",
-            "injuries_sustained",
-            "loss_of_property",
-            "witnesses",
-            "submitted_by",
-            "police_abstract_obtained",
-        )
+        fields = '__all__'
```

Comment:
This isn't a secured way to expose all fields. It's more appropriate to list out the fields to serialize and de-serialize explicitly to avoid accidentally exposing a field which should be hidden.